language-service/language_service/dto/__init__.py

```diff
@@ -0,0 +1,29 @@
+class Definition:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in public package](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)